### ISSUE TITLE
Garage App on Phone

### ISSUE THREAD

**Issue by PeanutSyrup** [JustLazzy/qb-garages#1]
Hey Lazzy, 

Great work! This is fantastic :D Only issue I am having is that the cars don't show up in garage app. Also, something else I noticed is that sometimes the car will come out of the garage broken.

**Comment by JustLazzy** [comment]
I don't know what cause your first issue, because I've never seen that before, I will wait until other people complaining about first issue and I will start investigating.

for your second issue, I'm having that issue even with default qb-garages, what I'm doing is I put the car back into the garage, and take it out.

**Comment by Fistsofury** [comment]
I have the cars not showing up in the app issue to.

QB version built 3 days ago so brand new version, no changes in the qb-discord since then either.
![Screenshot 2022-03-22 120146](https://user-images.githubusercontent.com/92045818/159477560-926170f3-0505-48cf-8a70-2643b0d9c772.png)

**Comment by SoStressful** [comment]
I do not have the app issue.

**Comment by Hobby-Less** [comment]
I also get the phone issue.

**Comment by SoStressful** [comment]
> 

Buy a car first and store it this is what the app looks like when you dont have a car lmao

**Comment by Hobby-Less** [comment]
Yeah because we would be dumb enough to not double check that we have cars I have 4 cars on the database showing where it is parked but not showing on the phone please don't try and be a smart ass

**Comment by SoStressful** [comment]
> Yeah because we would be dumb enough to not double check that we have cars I have 4 cars on the database showing where it is parked but not showing on the phone please don't try and be a smart ass

I said "lmao" don't start crying ahahah

**Comment by Hobby-Less** [comment]
Ok buddy I am just here for a solution not to argue with someone that's not willing to help

**Comment by PeanutSyrup** [comment]
> I don't know what cause your first issue, because I've never seen that before, I will wait until other people complaining about first issue and I will start investigating.
> 
> for your second issue, I'm having that issue even with default qb-garages, what I'm doing is I put the car back into the garage, and take it out.

I haven't had the issue with default QB. Weird. So cars show up in garage app for you?

Something else that seems to be a minor issue would be the cars if impounded and then taken out of police impound are not owned by the individual. They can't grab it and drive it away and park it. If they try and park it in a garage it says they don't own it.

**Comment by PeanutSyrup** [comment]
> > 
> 
> Buy a car first and store it this is what the app looks like when you dont have a car lmao

I have purchased cars and same issue.

**Comment by JustLazzy** [comment]
> 

yep, its show up in my garage app like usual

**Comment by Kentainfr** [comment]
Make sure your phone is up to date, and if it doesn't work try this phone too : https://github.com/vBooya/qb-phone-npstyle

**Comment by PeanutSyrup** [comment]
> Make sure your phone is up to date, and if it doesn't work try this phone too : https://github.com/vBooya/qb-phone-npstyle

This was actually the fix. I had a couple month old phone but this fixed my issue. Thank you!!
